### DISCUSSION TITLE
How to remove the package completely?

### DISCUSSION THREAD

**Discussion by Cc0r** [z-shell/zi#316] [Q&A]
I tried to search on the website using different queries, but could not see how to do this, I only found the information how to uninstall plugin, but that was about it. You would expect uninstall option be a part of main application menu, but also it's not here. So please. fuzzers - give me some clues!

**Comment by topernic** [comment]
I am jumping on that bandwagon. Everything about installing, using, and plugins is there and it is great.
But I'm with you, there needs to documentation on removing it.
